### ISSUE TITLE
Implementation of strict mode

### ISSUE THREAD

**Issue by PINTO0309** [PINTO0309/onnx2tf#145]
### Issue Type

Others

### onnx2tf version number

1.7.x

### onnx version number

1.13.0

### tensorflow version number

2.10.0

### Download URL for ONNX

N/A

### Parameter Replacement JSON

```json
N/A
```

### Description

1. Personal
4. Add an ultra-slow conversion mode to strictly avoid shape unmatch errors and precision errors.
### Background
Since the internal processing has already implemented a number of fairly complex workarounds to avoid conversion errors, we have not dared to implement a correction process for accuracy errors at this time in order not to further increase the complexity of the logic. Instead, I have implemented a mechanism whereby the user visually identifies which operation's channel transpositions cause problems, and the user changes the behavior of the tool himself.

However, I cannot overlook the situation where accuracy errors remain despite successful model transformations, and I hope to eradicate them in the future. The work to change the behavior of the tool by the users themselves with the users' visibility of the problem areas is very costly.

Since the internal processing mechanism needs to be significantly revised, we intend to implement many additional test modifications gradually in minor version upgrades to the extent that they do not affect the existing processing.

### Idea
  - Add all OPs of ONNX to the output OP of the graph.
  - Run ONNX inference with dummy tensor only once to get the sample inference results for all OPs.
  - Always compare the contents of the ONNX output tensor and TensorFlow output tensor before converting each OP.
  - Channel placement will always be inconsistent between ONNX and TensorFlow, so the following implementation can be called from all OPs at any time.
    https://github.com/PINTO0309/onnx2tf/blob/33e65dcf25f60bfe83a7abf39349cf51425616f6/onnx2tf/utils/common_functions.py#L2937-L3062
  - Search for the channel arrangement with the lowest accuracy error.
  - The `Transpose` OP is excluded from the search because the value to be compared does not change.
  - Not implemented for `Constant` and `ConstantOfShape`.
  - Other than the above, OPs that do not involve conversion of values are excluded from processing.
  - For error correction, the same method as for `pre_process_transpose_perm` is used to check all combinations that pre-transpose the input tensor to the corresponding OP.
  - If the target OP is specified in `parameter_replacement.json`, omit processing.
  - Remove any intrinsic processing already implemented in `MatMul`.
  - OPs with both attribute and tensor values, such as `Softmax`, attempt correction in the following order: attribute value change first, then tensor transposition.

**Comment by PINTO0309** [comment]
I close because I now know how to address this issue.

- [[InSPyReNet] Swin Transformer Support question #312](https://github.com/PINTO0309/onnx2tf/issues/312)
- [[DN-DAB-DETR] The output of ONNX's Mul OP is different from the TFLite's output. #327](https://github.com/PINTO0309/onnx2tf/issues/327)
- [[TODO] Implement a process to reduce accuracy degradation due to transposition errors in the Transformer's MatMul input values. #317](https://github.com/PINTO0309/onnx2tf/issues/317)
